### PR TITLE
Cosmetics model

### PR DESCRIPTION
**Pull request by nikolatulechki**
Model based on first line of cosmetics pilot data.

### CODE REVIEW

File: data/Symbeeosis/codedLists.ttl [hunk 1]
```diff
@@ -0,0 +1,55 @@
+bdg:AssayCompound a rdfs:Class; rdfs:subClassOf skos:Concept;
+  rdfs:label "Assay compound codelist class".
+<assayCompound> a skos:ConceptScheme;
+  rdfs:label "Assay compound codelist scheme".
+<assayCompound/DPPH> a skos:Concept, bdg:AssayCompound;
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/codedLists.ttl:7]: add `rdfs:seeAlso` link

File: data/Symbeeosis/codedLists.ttl [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  skos:inScheme      <assayCompound>; skos:topConceptOf <assayCompound>;
+  skos:prefLabel     "DPPH" ;
+  skos:exactMatch       wd:Q5205953 .
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/codedLists.ttl:10]: Better use PubChem or CHEBI IDs, eg DPPH is
- https://pubchem.ncbi.nlm.nih.gov/compound/74358: rich html page
- http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID74358: RDF.
```ttl
@prefix pubchem: <http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID>.
```

File: data/Symbeeosis/codedLists.ttl [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+<assayCompound/ABTS> a skos:Concept, bdg:AssayCompound ;
+  skos:inScheme      <assayCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "ABTS" ;
+  skos:exactMatch       wd:Q287582 .
+<asseyCompound/Trolox> a skos:Concept, bdg:AssayCompound ;
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/codedLists.ttl]:
```suggestion
<assayCompound/Trolox> a skos:Concept, bdg:AssayCompound ;
```

typo

File: data/Symbeeosis/codedLists.ttl [hunk 4]
```diff
@@ -0,0 +1,55 @@
+bdg:AssayCompound a rdfs:Class; rdfs:subClassOf skos:Concept;
+  rdfs:label "Assay compound codelist class".
+<assayCompound> a skos:ConceptScheme;
+  rdfs:label "Assay compound codelist scheme".
+<assayCompound/DPPH> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <assayCompound>; skos:topConceptOf <assayCompound>;
+  skos:prefLabel     "DPPH" ;
+  skos:exactMatch       wd:Q5205953 .
+<assayCompound/ABTS> a skos:Concept, bdg:AssayCompound ;
+  skos:inScheme      <assayCompound>; skos:topConceptOf <asseyCompound>;
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/codedLists.ttl]:
```suggestion
  skos:inScheme      <assayCompound>; skos:topConceptOf <assayCompound>;
```

File: data/Symbeeosis/codedLists.ttl [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+<asseyCompound/GallicAcid> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <asseyCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "Gallic Acid" ;
+  skos:exactMatch       wd:Q375837 .
+<asseyCompound/Quercetin> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <asseyCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "Quercetin" ;
+  skos:exactMatch       wd:Q409478 .
+
+<extractionMethod> a skos:ConceptScheme;
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/codedLists.ttl]:
```suggestion
<preparationMethod> a skos:ConceptScheme;
```
These methods just break up the leaves, they don't extract any substance from the leaves

File: data/Symbeeosis/codedLists.ttl [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+<microbialCount> a skos:ConceptScheme;
+  rdfs:label "Microbial Count ranges codelist scheme" .
+bdg:MicrobialCount a rdfs:Class; rdfs:subClassOf skos:Concept;
+  rdfs:label "Microbial Count ranges codelist class" .
+<microbialCount/LT10> a skos:Concept , bdg:MicrobialCount ;
+  skos:prefLabel "< 10";
+  skos:inScheme <microbialCount> ; skos:topConceptOf <microbialCount> .
+
+<grapeVariety> a skos:ConceptScheme;
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/codedLists.ttl:51]: Ask the agricultural researchers about the relation between vinestock, variety, and wine.
- maybe a wine can be made from several varieties, so it should say which varieties (and in what proportion?) have gone into making it
- are there more levels, eg sort, varietal...

File: data/Symbeeosis/observations.ttl [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+<data/cosmetics> a qb:DataSet ;
+    rdfs:label    "Measurement  for cosmetics pilot";
+    qb:structure  <DSD/cosmetics> ;
+    sosa:hasFeatureOfInterest <feature/VineLeaves> .
+
+<DSD/cosmetics> a qb:DataStructureDefinition ;
+    qb:component
+    [qb:attribute sosa:hasFeatureOfInterest  ; qb:componentAttachment qb:DataSet],
+    [qb:attribute sdmx-attribute:unitMeasure ; qb:componentAttachment qb:MeasureProperty],
+    [qb:attribute bdg:assayMethod ; qb:componentAttachment qb:MeasureProperty ],
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/observations.ttl]:
```suggestion
    [qb:attribute bdg:assayMethod ;              qb:componentAttachment qb:MeasureProperty ],
```
align

Comment (VladimirAlexiev) [inline data/Symbeeosis/observations.ttl]: Make `assayMethod, comparisonBaseline` Optional: `componentRequired false`

File: data/Symbeeosis/observations.ttl [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    [qb:measure   bdg:refractiveIndex ],
+    [qb:measure   bdg:yeastsAndMoulds ],
+    [qb:measure   bdg:totalMicrobialCount ],
+    [qb:measure   bdg:antioxidantActivityDPPHTrolox ],
+    [qb:measure   bdg:antioxidantActivityABTSTrolox ],
+    [qb:measure   bdg:TPCGallicAcid ],
+    [qb:measure   bdg:TFCQuercetin ].
+
+
+<data/cosmetics/IA1U> a qb:Obesrvation ;
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/observations.ttl]:
```suggestion
<data/cosmetics/IA1U> a qb:Observation ;
```

File: data/Symbeeosis/observations.ttl [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    qb:dataSet                          <data/cosmetics> ;
+    bdg:extractionMethod                <extractionMethod/Ultrasound>;
+    bdg:grapeVariety                    <grapeVariety/Agiorgitiko>;
+    bdg:date                            2018-05-08^^xsd:date;
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/observations.ttl:30]:
```suggestion
    bdg:date                            "2018-05-08"^^xsd:date;
```

File: data/Symbeeosis/observations.ttl (hunk 8 above)
Comment (VladimirAlexiev) [inline data/Symbeeosis/observations.ttl]: 1. `sample` is missing

File: data/Symbeeosis/observations.ttl [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+    qb:structure  <DSD/cosmetics> ;
+    sosa:hasFeatureOfInterest <feature/VineLeaves> .
+
+<DSD/cosmetics> a qb:DataStructureDefinition ;
+    qb:component
+    [qb:attribute sosa:hasFeatureOfInterest  ; qb:componentAttachment qb:DataSet],
+    [qb:attribute sdmx-attribute:unitMeasure ; qb:componentAttachment qb:MeasureProperty],
+    [qb:attribute bdg:assayMethod ; qb:componentAttachment qb:MeasureProperty ],
+    [qb:attribute bdg:comparisonBaseline ; qb:componentAttachment qb:MeasureProperty ],
+    [qb:dimension bdg:extractionMethod ],
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/observations.ttl:12]: Consider moving to a slice: `qb:componentAtachment qb:Slice`

File: data/Symbeeosis/variables.ttl [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+  skos:description "pH";
+  qudt:hasQuantityKind       quantitykind:Dimensionless;
+  sdmx-attribute:unitMeasure unit:NUM .
+
+bdg:refractiveIndex a qb:MeasureProperty, sosa:ObservableProperty;
+  rdfs:label                 "Refractive Index";
+  skos:notation              "refractiveIndex";
+  skos:description           "Measurement of brix percent of the liquid solution";
+  sdmx-attribute:unitMeasure unit:PERCENT ;
+
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/variables.ttl:15]:
```suggestion
.
```

File: data/Symbeeosis/variables.ttl [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+bdg:totalMicrobialCount a qb:MeasureProperty, qb:CodedProperty, sosa:ObservableProperty;
+  qb:codeList                <microbialCount> ;
+  rdfs:range                 bdg:MicobialCount ;
+  skos:notation              "totalMicrobialCount";
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/variables.ttl:20]: add rdfs:label. (and I don't know what we need `notation` for)

File: data/Symbeeosis/variables.ttl [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+  skos:description           "Total microbial count: Measurement of TPC with classic development of micro-organism in  petri-dishes";
+
+bdg:yeastsAndMoulds a qb:MeasureProperty, qb:CodedProperty, sosa:ObservableProperty;
+  qb:codeList                <microbialCount> ;
+  rdfs:range                 bdg:MicobialCount ;
+  skos:notation              "yeastsAndMoulds";
+  skos:description           "Measurement of Y&M with classic development of micro-organism in petri dishes";
+
+bdg:antioxidantActivityDPPHTrolox a qb:MeasureProperty, sosa:ObservableProperty;
+  rdfs:label                 "Antioxidant activity DPPH";
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/variables.ttl]: label must mention Trolox

File: data/Symbeeosis/variables.ttl [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+bdg:antioxidantActivityABTSTrolox a qb:MeasureProperty, sosa:ObservableProperty;
+  rdfs:label                 "Antioxidant activity ABTS";
+  skos:notation              "antioxidantActivityABTSTrolox";
+  skos:description           "Antioxidant activity: Spectrophotometric method of antioxidant capacity using ABTS assay measuring density of trolox equivalent" ;
+  sdmx-attribute:unitMeasure bdg-unit:UG-PER-ML ;
+  bdg:assayMethod            <assayCompound/ABTS> ;
+  bgd:comparisonBaseline     <assayCompound/Trolox> .
+
+bdg:TPCGallicAcid a qb:MeasureProperty, sosa:ObservableProperty;
+  rdfs:label                 "Total phenolic content";
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/variables.ttl]: add "... compared to Gallic Acid"

File: data/Symbeeosis/variables.ttl [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+  skos:notation              "TPCGallicAcid";
+  skos:description           "Spectrophotometric measurement of the phenolic content in the extract using TPC assay using gallic acid as a comparison baseline" ;
+  sdmx-attribute:unitMeasure bdg-unit:UG-PER-ML ;
+  bgd:comparisonBaseline     <assayCompound/GallicAcid> .
+
+bdg:TFCQuercetin a qb:MeasureProperty, sosa:ObservableProperty;
+  rdfs:label                 "Total flavonoid content";
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/variables.ttl]: add "... compared to Quercetin"

File: data/Symbeeosis/variables.ttl [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+  skos:notation              "yeastsAndMoulds";
+  skos:description           "Measurement of Y&M with classic development of micro-organism in petri dishes";
+
+bdg:antioxidantActivityDPPHTrolox a qb:MeasureProperty, sosa:ObservableProperty;
+  rdfs:label                 "Antioxidant activity DPPH";
+  skos:notation              "antioxidantActivityDPPHTrolox";
+  skos:description           "Antioxidant activity: Spectrophotometric method of antioxidant capacity using DPPH assay measuring density of trolox equivalent" ;
+  sdmx-attribute:unitMeasure bdg-unit:UG-PER-ML ;
+  bdg:assayMethod            <assayCompound/DPPH> ;
+  bgd:assessedCompound       <assayCompound/Trolox> .
```
Comment (VladimirAlexiev) [inline data/Symbeeosis/variables.ttl]:
```suggestion
  bgd:comparisonBaseline   <assayCompound/Trolox> .
```

File: model/units.ttl [hunk 17]
```diff
@@ -0,0 +1,39 @@
+bdg-unit:UG-PER-ML
+  rdf:type qudt:DerivedUnit ;
+  rdf:type qudt:MassPerVolumeUnit ;
+  rdf:type qudt:Unit ;
+  qudt:abbreviation "μg/mL" ;
+  qudt:conversionMultiplier 1.0e0 ;
+  qudt:conversionOffset "0.000001"^^xsd:double ;
```
Comment (VladimirAlexiev) [inline model/units.ttl]:
```suggestion
  qudt:conversionOffset 0.0 ;
```

File: model/units.ttl [hunk 18]
```diff
@@ -0,0 +1,39 @@
+bdg-unit:UG-PER-ML
+  rdf:type qudt:DerivedUnit ;
+  rdf:type qudt:MassPerVolumeUnit ;
+  rdf:type qudt:Unit ;
+  qudt:abbreviation "μg/mL" ;
+  qudt:conversionMultiplier 1.0e0 ;
```
Comment (VladimirAlexiev) [inline model/units.ttl]:
```suggestion
  qudt:conversionMultiplier 1e-6 ;
```

File: model/units.ttl [hunk 19]
```diff
@@ -0,0 +1,39 @@
...
+  qudt:conversionOffset "0.0"^^xsd:double ;
+  qudt:description "Gram per liter is a convenience unit, numerically the same as Kilogram per Cubic Meter" ;
+  qudt:hasQuantityKind quantitykind:Density ;
+  qudt:symbol "g l^{-1}" ;
+  rdfs:isDefinedBy bdg-unit: ;
+  rdfs:label "Gram per Liter" ;
+  skos:prefLabel "gram-per-liter" ;
+.
+
+bgd-unit DEG_BRIX
```
Comment (VladimirAlexiev) [inline model/units.ttl]:
```suggestion
bgd-unit:DEG_BRIX
```

File: model/units.ttl [hunk 20]
```diff
@@ -0,0 +1,39 @@
...
+  rdf:type qudt:Unit;
+  rdfs:label "Degree Brix";
+  skos:prefLabel "Degree-brix" ;
+  qudt:abbreviation "°Brix";
+  qudt:conversionMultiplier 1.0e0 ;
+  qudt:conversionOffset "0.0"^^xsd:double ;
+  qudt:description "A Brix degree equals 1 gram of sucrose per 100 grams of solution." ;
+.
```
Comment (VladimirAlexiev) [inline model/units.ttl:42]: - "This is a convenience unit equal to percent, but applicable only to this specific compound".
- add some characteristics of unit:PERCENT, eg kind=Dimensionless
- if possible, relate to unit:PERCENT

File: model/units.ttl [hunk 21]
```diff
@@ -0,0 +1,39 @@
...
+  qudt:abbreviation "μg/mL" ;
+  qudt:conversionMultiplier 1.0e0 ;
+  qudt:conversionOffset "0.000001"^^xsd:double ;
+  qudt:description "Microgram per milliliter is KG-PER-M3 scaled by 10^{-6}" ;
+  qudt:hasQuantityKind quantitykind:Density ;
+  qudt:symbol "μg/cm^3" ;
+  rdfs:isDefinedBy bdg-unit: ;
+  rdfs:label "Microgram per Milliliter" ;
+  skos:prefLabel "microgram-per-milliliter" ;
+.
```
Comment (VladimirAlexiev) [inline model/units.ttl:15]:
```ttl
skos:exactMatch obo:UO_0000274;
```

**Review by VladimirAlexiev — CHANGES_REQUESTED** [review]

File: data/Symbeeosis/observations.ttl (hunk 7 above)
Comment (nikolatulechki) [reply data/Symbeeosis/observations.ttl]: Added it, but default is false

**Pull request merged**